### PR TITLE
Fix travis-ci badge path in the main README.md

### PR DESCRIPTION
**Pull request by vors**
Fixing this
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -312,5 +312,5 @@ providers to supply override values to the component.
 [doc-img]: https://godoc.org/go.uber.org/fx?status.svg
 [cov]: https://coveralls.io/github/uber-go/uberfx?branch=master
 [cov-img]: https://coveralls.io/repos/github/uber-go/uberfx/badge.svg?branch=master
```
Comment (glibsm) [inline README.md]: Do these need to change as well?

**Comment by glibsm** [comment]
Thank you for the PR, just one question before merge. 

We renamed the repository today and are still catching up with all the things that need to be done along with it

File: README.md (hunk 1 above)
Comment (vors) [inline README.md]: Likely

**Comment by sectioneight** [comment]
Thanks @vors, this totally slipped my mind when renaming yesterday.

**Pull request merged**